### DISCUSSION TITLE
Incorrect extraction in tables

### DISCUSSION THREAD

**Discussion by tujinshu** [jsvine/pdfplumber#923] [Ask for help with specific PDFs]
## Describe the bug

*A clear and concise description of what the bug is.*
table extract error in pdf，one colum extract to several  colum

## Code to reproduce the problem

*Paste it here, or attach a Python file.*
```
import pdfplumber
pdf = pdfplumber.open("./wps.pdf")
p0 = pdf.pages[168]
im = p0.to_image()
im.debug_tablefinder()
```

## PDF file
[wps.pdf](https://github.com/jsvine/pdfplumber/files/11943505/wps.pdf)

## Expected behavior
page 169 origin content:
![image](https://github.com/jsvine/pdfplumber/assets/7986082/427c289d-4544-4376-b492-911cf065ade5)

## Actual behavior

![image](https://github.com/jsvine/pdfplumber/assets/7986082/0511d126-782c-4bb3-8961-fe960da904ed)

## Screenshots

![image](https://github.com/jsvine/pdfplumber/assets/7986082/0511d126-782c-4bb3-8961-fe960da904ed)

## Environment

- pdfplumber version: [0.9.0]
- Python version: [3.8.0]
- OS: [ Linux]

## Additional context

*Add any other context/notes about the problem here.*

**Comment by jsvine** [comment]
Hi @tujinshu. Does my response to your other question — https://github.com/jsvine/pdfplumber/discussions/924 — help you solve this one too?

**Reply by tujinshu** [reply]
thank you for your reply，i will have try！
